cmd/commands/repo.go

```diff
@@ -165,7 +165,7 @@ func NewRepoBootstrapCommand() *cobra.Command {
 
 	<BIN> repo bootstrap --repo https://github.com/example/repo/path/to/installation_root
 `),
-		PreRun: func(_ *cobra.Command, _ []string) { cloneOpts.Parse() },
+		PreRun: func(cmd *cobra.Command, args []string) { cloneOpts.Parse() },
```

Comment:
why this change?

Reply:
It was probably caused by the revert. I'll revert the revert